modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase

```diff
@@ -1139,6 +1139,59 @@ definition {
 			value1 = "Showing 1 to 2 of 2");
 	}
 
+	@description = "COMMERCE-7198. I want to be able to search for one pending order using the search bar on the Account Selector"
+	@priority = 4
```

Comment:
As the test will be included in the relevant, please change the priority from 4 to 5